Purchases/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -32,111 +32,88 @@ class SubscriberAttributesManager {
     }
 
     func setAttributes(_ attributes: [String: String], appUserID: String) {
-        logAttributionMethodCalled(functionName: #function)
+        Logger.debug(Strings.attribution.setting_attributes(attributes: Array(attributes.keys)))
         for (key, value) in attributes {
             setAttribute(key: key, value: value, appUserID: appUserID)
         }
     }
 
     func setEmail(_ maybeEmail: String?, appUserID: String) {
-        logAttributionMethodCalled(functionName: #function)
```

Comment:
it felt odd that all methods were making the same 2 calls, so I moved this call into the `setSpecialAttribute` method

Reply:
I'm also not passing in the function and updated the log a bit. 
| Before | After |
| :-: | :-: |
| `"setAirshipChannelID called"` | `"setting special attribute: $airshipChannelID"` |